### PR TITLE
Update dashboard to 2.5

### PR DESCRIPTION
**Pull request by lucferbux**
- [X] The commits are squashed in a cohesive manner and have meaningful messages.

### CODE REVIEW

**Review by andrewballantyne — APPROVED** [review]

File: odh-dashboard/modelserving/caikit-ootb.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  labels:
+    opendatahub.io/dashboard: 'true'
+  spec:
+    multiModel: false
+    supportedModelFormats:
+      - autoSelect: true
+        name: caikit
+    containers:
+      - name: kserve-container
+        image: text-generation-inference
```
Comment (bdattoma) [inline odh-dashboard/modelserving/caikit-ootb.yaml:32]: quay.io/opendatahub/text-generation-inference:fast

File: odh-dashboard/modelserving/caikit-ootb.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        command:
+          - text-generation-launcher
+        args:
+          - --model-name=/mnt/models/artifacts/
+        env:
+          - name: TRANSFORMERS_CACHE
+            value: /tmp/transformers_cache
+      - name: transformer-container
+        image: caikit-tgis-serving
```
Comment (bdattoma) [inline odh-dashboard/modelserving/caikit-ootb.yaml:41]: quay.io/opendatahub/caikit-tgis-serving:fast

File: odh-dashboard/modelserving/caikit-ootb.yaml (hunk 1 above)
Comment (andrewballantyne) [reply odh-dashboard/modelserving/caikit-ootb.yaml:32]: These have always been environment variables passed to Kustomize... @lucferbux are these handled?

Comment (lucferbux) [reply odh-dashboard/modelserving/caikit-ootb.yaml:32]: Yes, this is kustomization image replacements, the sha is in the bas kustomize file in the PR.

File: odh-dashboard/modelserving/caikit-ootb.yaml (hunk 2 above)
Comment (lucferbux) [reply odh-dashboard/modelserving/caikit-ootb.yaml:41]: Same here

File: odh-dashboard/modelserving/caikit-ootb.yaml (hunk 1 above)
Comment (bdattoma) [reply odh-dashboard/modelserving/caikit-ootb.yaml:32]: ok sorry then :)

**Comment by lucferbux** [comment]
@VaishnaviHire @zdtsw can someone merge this? it would be needed for 2.6

**Pull request merged**